lambdas/data-migration1/src/collections.ts

```diff
@@ -25,20 +25,30 @@ export const migrateCollectionRecord = async (
   dynamoRecord: AWS.DynamoDB.DocumentClient.AttributeMap,
   knex: Knex
 ): Promise<void> => {
-  const existingRecord = await knex<PostgresCollectionRecord>('collections')
-    .where({
+  const collectionPgModel = new CollectionPgModel();
+
+  let existingRecord;
+
+  try {
+    existingRecord = await collectionPgModel.get(knex, {
       name: dynamoRecord.name,
       version: dynamoRecord.version,
-    })
-    .first();
+    });
+  } catch (error) {
+    // Swallow any RecordDoesNotExist errors and proceed with migration,
```

Comment:
Nit: this comment seems clear from the code.

Reply:
Yeah, that's fair. It also doesn't explain WHY we're doing that so it's a pretty useless comment.

https://github.com/nasa/cumulus/pull/2162/commits/197e285aba1cb3a18dd5986dc5a2662455b6a453